src/main/controllers/SubscriptionUrnSearchResultController.ts

```diff
@@ -1,15 +1,17 @@
 import { Response } from 'express';
 import { PipRequest } from '../models/request/PipRequest';
-import { SubscriptionService } from '../service/subscriptionService';
+
 import { cloneDeep } from 'lodash';
+import { HearingService } from '../service/hearingService';
 
-const subscriptionService = new SubscriptionService();
+const hearingService = new HearingService();
 
 export default class SubscriptionUrnSearchResultController {
   public async get(req: PipRequest, res: Response): Promise<void> {
     const searchInput = req.query['search-input'];
     if (searchInput && searchInput.length) {
-      const searchResults = await subscriptionService.getSubscriptionUrnDetails(searchInput.toString());
+
```

Comment:
?

Reply:
not sure but there is a line not showing here

Reply:
you mean theres code that isnt showing? or you dont know why you added the line?

Reply:
there is code not showing. the line should be 
const searchResults = await hearingService.getCaseByURN(searchInput.toString());